src/app/components/demo-page.component.html

```diff
@@ -15,9 +15,13 @@
   <stache-page-anchor>
     Code
   </stache-page-anchor>
-  <div class="stache-demo-code" *ngFor="let file of demoFiles">
-    <h4>{{file.name}}</h4>
-    <stache-code-block code="{{file.fileContents}}" languageType="{{file.language}}">
-    </stache-code-block>
-  </div>
+  <sky-tabset>
```

Comment:
Would be helpful to wrap the entire thing in a reusable component, and hide all the complex functionality from the consumer, which would use SKY Tabs, set active states, etc.
```
<stache-code-samples>
  <stache-code-sample>
    <stache-code-sample-heading>
      index.html
    </stache-code-sample-heading>
    <stache-code-block>
      <h1>This is my code sample...</h1>
    </stache-code-block>
  </stache-code-sample>
</stache-code-samples>
```